src/app/devices/deviceContent/components/deviceSettings/deviceSettingsPerInterfacePerSetting.tsx

```diff
@@ -168,13 +167,15 @@ export default class DeviceSettingsPerInterfacePerSetting
         const { reportedTwin, settingModelDefinition : modelDefinition, settingSchema : schema } = this.props;
         return (
             <div role="dialog">
```

Comment:
I think it would be better to move the `{this.state.showReportedValuePanel &&` up one line to the `<div role="dialog">` so that the entire component is on or off. You might need to include a `||<></>`.